src/config/config.py

```diff
@@ -58,6 +58,8 @@
     "endoflife",
     "reconnaissance",
 ]
+LAST_QUERY_ID = 16
```

Comment:
I'm concerned that when someone adds a new query to the query library, they might forget to update the constant. As I see it, we have two options: make this constant a dynamic variable that reflects the number of query files in the query directory, or introduce a test to ensure the number of queries in the directory matches the constant. 
What are your thoughts?